### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-python-library.git`](https://github.com/cisagov/skeleton-python-library.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/assessment-data-export.git assessment-data-export
    cd assessment-data-export
    git remote add skeleton https://github.com/cisagov/skeleton-python-library.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add README.md src/ade/_version.py 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -12,9 +12,16 @@ updates:
...
+      # Managed by cisagov/skeleton-generic
       - dependency-name: actions/cache
       - dependency-name: actions/checkout
+      - dependency-name: actions/setup-go
       - dependency-name: actions/setup-python
+      - dependency-name: hashicorp/setup-terraform
+      - dependency-name: mxschmitt/action-tmate
+      # # Managed by cisagov/skeleton-python-library
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/upload-artifact
```
Comment (jsf9k) [inline .github/dependabot.yml]:
```suggestion
      # Managed by cisagov/skeleton-python-library
      - dependency-name: actions/download-artifact
      - dependency-name: actions/upload-artifact
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
Please correct the small oversight I commented on.

Also, here is a friendly reminder to create a release after merging:  :penguin:

File: .github/dependabot.yml (hunk 1 above)
Comment (jsf9k) [reply .github/dependabot.yml]: FWIW, I have been doing a `git diff origin/develop` after resolving the conflicts (but before force-pushing) to look for these sorts of oversights.

Comment (dav3r) [reply .github/dependabot.yml]: Thanks - I knew I was forgetting something last night. 👍

File: setup.py [hunk 2]
```diff
@@ -97,6 +100,7 @@ def get_version(version_file):
     ],
     extras_require={
         "test": [
+            "boto3-stubs",
```
Comment (mcdonnnj) [inline setup.py]: Are these typing packages actually necessary for testing? The `mypy` package is only installed as part of the development requirements so I think these should go in `requirements-dev.txt`.

File: src/ade/assessment_data_export.py [hunk 3]
```diff
@@ -43,10 +43,16 @@
...
+# I cannot find a package that includes type stubs for defusedxml, so use
+# "type: ignore" to tell mypy to ignore this library
+from defusedxml import ElementTree  # type: ignore
 from docopt import docopt
 import requests
-from xmljson import badgerfish as bf
+
+# I cannot find a package that includes type stubs for xmljson, so use
+# "type: ignore" to tell mypy to ignore this library
+from xmljson import badgerfish as bf  # type: ignore
```
Comment (mcdonnnj) [inline src/ade/assessment_data_export.py:55]: Good thing this package is dead 🙃 

> This library is not actively maintained. Alternatives are [xmltodict](https://github.com/martinblech/xmltodict) and [untangle](https://untangle.readthedocs.io/en/latest/). Use only if you need to parse using specific XML to JSON [conventions](http://wiki.open311.org/JSON_and_XML_Conversion/).

**Review by mcdonnnj — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (dav3r) [reply setup.py]: Makes sense - please see https://github.com/cisagov/assessment-data-export/pull/46/commits/799661f7438cdb6f782150e49d9079aecf11a140.

**Review by jsf9k — APPROVED** [review]

File: requirements-dev.txt [hunk 4]
```diff
@@ -1,4 +1,7 @@
 --requirement requirements-test.txt
+boto3-stubs
 ipython
 mypy
 semver
+types-docopt
+types-requests
```
Comment (mcdonnnj) [inline requirements-dev.txt:7]:
```suggestion
types-requests
types-setuptools
```

File: .pre-commit-config.yaml [hunk 5]
```diff
@@ -90,35 +90,37 @@ repos:
...
       - id: isort
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.942
+    rev: v0.991
     hooks:
       - id: mypy
         additional_dependencies:
+          - boto3-stubs
+          - types-docopt
           - types-requests
```
Comment (mcdonnnj) [inline .pre-commit-config.yaml:121]:
```suggestion
          - types-requests
          - types-setuptools
```

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔Just one pair of suggestions for your consideration.

**Pull request merged**